SQL_Assignments/sql_5.sql

```diff
@@ -0,0 +1,45 @@
+uber
+
+
+
+create table all_address(pincode int primary key,street varchar(25),city varchar(25),state_name varchar(25),country varchar(25))
+
+create table Customers(Customer_id int primary key,Customer_name varchar(25),Customer_email varchar(25) unique,customer_phone_no varchar(13),pincode int,foreign key(pincode) references all_address(pincode))
+
+create table Drivers(Driver_id int primary key,driver_name varchar(25),Driver_email varchar(25) unique,driver_phone_no varchar(13),pincode int,foreign key(pincode) references all_address(pincode))
+
+create table ride_details(Customer_id int,Driver_id int,Ride_date_time Datetime,amount decimal(10,2),payment_method varchar(25),foreign key(Driver_id) references Drivers(Driver_id),foreign key(Customer_id) references Customers(Customer_id))
```

Comment:
Not normalized

Reply:
updated the tables and new file uploaded